docs/library/collections.rst

```diff
@@ -39,7 +39,7 @@ Classes
         Remove and return an item from the left side of the deque.
         Raises IndexError if no items are present.
 
-.. function:: namedtuple(name, fields)
+.. class:: namedtuple(name, fields)
```

Comment:
This is actually a function that creates classes (CPython also documents this as a function).